applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.hpp

```diff
@@ -138,8 +138,14 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwBaseElement : public Element
 
     std::string Info() const override
     {
+        std::stringstream claw_buffer;
+        if (mConstitutiveLawVector.size() != 0) {
+            claw_buffer << mConstitutiveLawVector[0]->Info();
+        } else {
+            claw_buffer << "not defined";
+        }
```

Comment:
We would suggest a one-line solution here and using constitutive_info in line 148 (the same applies to the other places where you fixed this): 
```suggestion
        const std::string constitutive_info = !mConstitutiveLawVector.empty() ? mConstitutiveLawVector[0]->Info() : "not defined";
```